backend/dataall/core/stacks/services/stack_service.py

```diff
@@ -86,6 +86,16 @@ def save_describe_stack_task(session, environment, stack, target_uri):
         session.commit()
         return cfn_task
 
+    @staticmethod
+    def create_stack(environment_uri, target_type, target_uri):
+        with get_context().db_engine.scoped_session() as session:
+            return StackRepository.create_stack(
+                session=session,
+                environment_uri=environment_uri,
+                target_type='environment',
+                target_uri=environment_uri,
```

Comment:
can we use the input params `target_type` and `target_uri` instead of hard-coded `environment` and re-using `environment_uri` 

This way `create_stack()` can be generic

Reply:
done